Sources/Orbit/Foundation/CountryFlags/CountryFlags.swift

```diff
@@ -260,5 +260,9 @@ public extension CountryFlag {
         case za
         case zm
         case zw
+
+        public init(_ string: String) {
```

Comment:
This file is autogenerated, so the change will be lost the next time flags are changed - I suggest moving this to a private extension in `CountryFlag.swift`.

Reply:
Thx, updated 👍